<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wally::qtfront::question_view_sub::BlockListMimic</class>
 <widget class="QGroupBox" name="wally::qtfront::question_view_sub::BlockListMimic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Block List</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="layout_blocks"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_addTextBlock">
       <property name="text">
        <string>Add Text Block</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_addImageBlock">
       <property name="text">
        <string>Add Image Block</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
